approval_frame/settings.py

```diff
@@ -9,7 +9,10 @@
 SECRET_KEY = 'jndet@)ed9ge4+!k605hc+$0tjb8u5oe)pdjab)-b3lsot_xnf'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+# For development purposes, set the DEBUG_* variables to True in a 
+# local_settings.py file, which we .gitignore. This file should be added
+# in the same directory that contains this settings.py file.
+DEBUG = False
 TEMPLATE_DEBUG = DEBUG
```

Comment:
Noticed that Django requires the ALLOWED_HOSTS to be set with a list of hosts in order for it to run with DEBUG=False 

> python manage.py runserver
CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False

Would we be setting this variable in production ? In that case, I could remove the DEBUG_* variables from settings.py altogether and have only be set in local_settings.py by the developer.

Alternatively we could add ALLOWED_HOSTS = [ "localhost" ] in the settings.py file which could be overwritten in production. 

Thanks!